commands/a_start_commands/start-mentor-room.js

```diff
@@ -30,16 +30,37 @@ module.exports = class StartMentors extends PermissionCommand {
      */
     async runCommand(message) {
 
+        let joinTicketEmoji = await Prompt.reactionPrompt('What is the join ticket emoji?', message.channel, message.author.id);
```

Comment:
Would be good to add some whitespace and comments to explain what is going on here.